src/frame/mod.rs

```diff
@@ -12,7 +12,80 @@ use std::{borrow::Cow, error, fmt};
 use crate::bytes::Bytes;
 
 /// A Modbus function code is represented by an unsigned 8 bit integer.
-pub type FunctionCode = u8;
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum FunctionCode {
+    ReadCoils,                  //=> 0x01,
```

Comment:
These comments introduce unneeded redundancies and do not provide any value.

Reply:
I've removed the Function code enum